calculatorUICOOP/Views/MainPage.xaml

```diff
@@ -2,38 +2,44 @@
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:d="http://xamarin.com/schemas/2014/forms/design"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:local="clr-namespace:calculatorUICOOP.ViewModels"
              mc:Ignorable="d"
-             x:Class="calculatorUICOOP.MainPage">
+             x:Class="calculatorUICOOP.MainPage"
+            
+          >
+    <ContentPage.BindingContext>
```

Comment:
Make this "ignorable" (change it to `<d:ContentPage.BindingContext>`). You're instantiating the view-model twice: Once here and the other in the code-behind.